concent_api/core/payments/bankster.py

```diff
@@ -241,7 +241,6 @@ def finalize_payment(deposit_claim: DepositClaim) -> Optional[str]:
         # The code below is executed in another transaction, so - in theory - deposit_claim object could be modified in
         # the meantime. Here we are working under assumption that it's not the case and it is coder's responsibility to
         # ensure that.
-        # Bankster puts transaction ID in DepositClaim.tx_hash.
```

Comment:
This commit says that you "Removed nested transactions in Bankster" but it only removes this comment.

Reply:
Originally it was, but after merging https://github.com/golemfactory/concent/pull/1041/ which has same changes this is only what is left from this commit.